doc/theory/glossary.tex

```diff
@@ -213,10 +212,19 @@
 	\begin{equation*}
 	||\lambda||_p = \left( \sum_{i \in \mathbb N} ||\lambda_i||^p_p \right)^{1/p}
 	\end{equation*}
-	whenever the right hand side exists and is finite. 
+	whenever the right hand side exists and is finite.
 	\paragraph{References:} \cite{stein2011functional, bubenik2015statistical}
+
+	\subsection*{Weighted silhouettes:}
+	Let $D = \{(b_i, d_i)\}_{i \in I}$ be the persistence diagram of a \underline{persistence module}. A \textit{weighted silhouette} associated to $D$ is a continuous function $\phi : \mathbb R \to \mathbb R$ of the form
+	\begin{equation*}
+	\phi(t) = \frac{\sum_{i\in I}w_i \Lambda_i(t)}{\sum_{i\in I}w_i},
```

Comment:
i \in I instead of i\in I ?

Reply:
Sure, though the LaTeX compiler doesn't see these things AFAIK.

Reply:
Fixed.